server/lists/index.js

```diff
@@ -293,7 +293,7 @@ module.exports = {
         },
         DRAFT_TEAMS: {
             client: 'INFO',
-            endpoint: '/teams/drafters',
```

Comment:
Is there a `/teams/drafters` endpoint we should tear out?

Reply:
ah, you already have. Don't mind me

Reply:
You're absolutely right. See here:
https://github.com/UKHomeOffice/hocs-info-service/pull/306/files